cartography/intel/aws/ec2/security_groups.py

```diff
@@ -146,6 +148,7 @@ def cleanup_ec2_security_groupinfo(neo4j_session: neo4j.Session, common_job_para
         neo4j_session,
         common_job_parameters,
     )
+    GraphJob.from_node_schema(EC2SecurityGroupSchema(), common_job_parameters).run(neo4j_session)
```

Comment:
We still need to keep both, for now, right?

Reply:
Yes this is intentional because the other resources cleaned up in aws_import_ec2_security_groupinfo_cleanup.json have not been refactored to the new model yet.